ntoskrnl/include/internal/i386/ke.h

```diff
@@ -722,7 +722,11 @@ KiSystemCallTrampoline(IN PVOID Handler,
         : "c"(StackBytes),
           "d"(Arguments),
           "r"(Handler)
+#ifndef __clang__
         : "%esp", "%esi", "%edi"
```

Comment:
I wonder how this can actually work. Calling the handler could (and most likely will) clobber ecx, edx & eax, and those are not in the list of clobbered registers.

Reply:
You can drop this commit & rebase on master. Thanks for pointing how broken this was.

Reply:
> 
> 
> You can drop this commit & rebase on master. Thanks for pointing how broken this was.

Done.